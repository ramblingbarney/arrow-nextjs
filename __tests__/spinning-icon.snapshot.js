import { render } from "@testing-library/react";
import SpinningIcon from "../app/components/spinning-icon";

describe("Page Snapshot", () => {
  it("renders homepage unchanged", () => {
    const { container } = render(<SpinningIcon />);
    expect(container).toMatchSnapshot();
  });
});
