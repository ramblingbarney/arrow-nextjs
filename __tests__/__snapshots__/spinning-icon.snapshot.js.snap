// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Snapshot renders homepage unchanged 1`] = `
<div>
  <div
    class="logo"
  >
    <img
      alt="React logo"
      class="logo react logo-clockwise"
      data-nimg="1"
      decoding="async"
      height="50"
      style="color: transparent;"
      width="50"
    />
    <h1
      class="title"
    >
      Time Mouse Stationary
       
      <span>
        0
      </span>
    </h1>
  </div>
</div>
`;
